spec/spec.ddoc

```diff
@@ -3,6 +3,36 @@ ROOT = ..
 SEARCHDEFAULT_SPEC = selected
 DDOC_BLANKLINE = $(DIVC blankline)
 
+BODY_PREFIX =
```

Comment:
This is a huge hack against Ddoc's capabilities. Maybe at some point in time we can switch to a proper template engine (or at least Ddox) ...